src/componentLibrary/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,31 @@
+import { CTA } from '../CTA';
+
+import { InfoBoxWrapper, InfoBoxHeader, InfoBoxText } from './InfoBox.style';
+
+interface InfoBoxProps {
+  title: string;
+  text: string;
```

Comment:
Thanks Peter — yes this is deliberate :) I want to limit this to a string because of the UX. If we allow other content we'll need to make some layout changes.